shared/js/background/atb.js

```diff
@@ -175,6 +182,11 @@ const ATB = (() => {
                         return !!atb
                     })
 
+                    // in case there is no assigned atb variant, enroll into Chrome Counteract experiment
+                    if (!atb && settings.getSetting(atb) && settings.getSetting(atb).atb && utils.getBrowserName() === 'chrome') {
```

Comment:
```suggestion
                    if (!atb && settings.getSetting('atb') && utils.getBrowserName() === 'chrome') {
```
You were calling `settings.getSetting(undefined)` which actually returns an object with all settings values. You should instead just ask specifically for the 'atb' setting: settings.getSetting('atb')

Reply:
ugh, my mistake, should be fixed now